src/v2/Apps/Partner/PartnerApp.tsx

```diff
@@ -24,12 +25,16 @@ export const PartnerApp: React.FC<PartnerAppProps> = ({
     displayFullPartnerPage,
     isDefaultProfilePublic,
     partnerPageEligible,
+    categories,
   } = partner
 
   if (!isDefaultProfilePublic || !partnerPageEligible) {
     throw new HttpError(404)
   }
 
+  const isBlackOwned =
+    categories!.filter(c => c && c.name === "Black Owned").length > 0
```

Comment:
I think it'd be useful to expose some field on Metaphysics for this. Like presumably there could be other ribbon labels for different kinds of galleries? Could there be a gallery that has more than one?

Reply:
In the spirit of impact over perfection we are embracing the single category haha. Its certainly possible in the future that we'll want to apply multiple categories and have them show up but we have no comps or concrete plans for this!